magento-research/pwa-studio
# Migrate off circleci

## Description
This PR is to follow #1087 . After confirming that the AWS CI checks are working as expected, then the follow up is to remove CircleCI configuration and enable blocking checks on the AWS codebuild checks. This PR also removes all CI configuration files from the repository and houses them in a private repository under the `magento` github organization. The same pr validation checks remain in this repository.

## Related Issue
Closes #1131 .

## Verification Steps
In this pull request we should see only AWS checks running and passing, once the required checks have been shifted from CircleCI to AWS.

items to check for feature parity with circleci checks working as expected:
- [ ] Coveralls reporting coverage in the comments
- [ ] DangerJS reporting in the comments when there is a failure (I can demo a test failure if needed)
- [ ] BundleSize reporting in the status checks
- [ ] AWS pwa-pr-build passing
- [ ] AWS pws-pr-test passing (which contains all of the checks that CircleCI was performing)
